tests/shtest

```diff
@@ -384,6 +384,60 @@ if [ "$($VALGRIND $Q $JQ -n '{"a":"xyz"} | halt_error(1)' 2>&1)" != '{"a":"xyz"}
     exit 1
 fi
 
+# Check binary (base64)
+$VALGRIND $Q $JQ -n 'tobinary("foo", 128, 129)' > $d/out
+printf '"Zm9vgIE="\n' > $d/expect
+cmp $d/out $d/expect
+
+$VALGRIND $Q $JQ -n 'tobinary("foo", 128, 129)' | $JQ -j --raw-output-binary '@base64dbinary' > $d/out
+printf 'foo\200\201' > $d/expect
+cmp $d/out $d/expect
+
+if [ "$(uname)" != "Darwin" ] && which base64; then
```

Comment:
FYI, `base64` and `base64 --decode` (reading stdin and writing stdout) should reliably work on macOS and GNU/Linux.

(I haven't tried the latter myself on macOS, but it should work based on man pages I've seen online).

Note that the `base64 $d/expect` below would need to be changed to `base64 < $d/expect`.